.github/workflows/trivy.yml

```diff
@@ -0,0 +1,24 @@
+name: Vulnerability scan
+
+on:
+  schedule:
+  - cron: "44 16 * * *"
+  push:
+  pull_request:
+
+jobs:
+  build:
+    name: Trivy fs scan
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v4
+    - name: Run trivy in fs mode
+      uses: aquasecurity/trivy-action@master
+      with:
+        scan-type: 'fs'
```

Comment:
Why would you scan just the fs? And not the Java components, for example? Why not a complete scan?

Reply:
I think this is the right type of scan for local repositories and that's what I've seen being used. 
In this case (of a maven project) trivy will look at the pom.xml inside the project directory and list vulnerabilities in found dependencies. Every component that is being used is listed in the pom file so there should be no need to check anything else.